pom.xml

```diff
@@ -4,7 +4,7 @@
     <artifactId>transferzero-sdk-java8</artifactId>
     <packaging>jar</packaging>
     <name>transferzero-sdk-java8</name>
-    <version>1.23.4-SNAPSHOT</version>
+    <version>1.23.5-SNAPSHOT</version>
```

Comment:
Is this correct?

Reply:
Yeah. I had issues releasing this SDK and so it went on by two versions. Also, having the `-SNAPSHOT` next to the version should mean that this version is still under development and so not yet released. It should not really have any major impact either ways.

Reply:
The README has `.4` though, shouldn't they be the same?